src/diet.ts

```diff
@@ -612,37 +635,56 @@ export function runDiet(): void {
             }
           }
           consumeSafe(countToConsume, menuItem.item);
-        } else if (menuItem.item === Mayo.flex) {
-          if (menuItems[menuItems.length - 1].item === $item`jumping horseradish`) {
-            MayoClinic.setMayoMinder(Mayo.zapine, countToConsume);
-          } else {
-            MayoClinic.setMayoMinder(Mayo.flex, countToConsume);
-          }
+        } else if ([Mayo.flex, Mayo.zapine].includes(menuItem.item)) {
```

Comment:
I know that we don't currently support mayodiol, but you might do like `Array.from(Object.entries(Mayo)).includes(menuItem.item)` if you feel like being terrible